runtime/syntax/man.vim

```diff
@@ -18,6 +18,11 @@ highlight default link manOptionDesc     Constant
 highlight default link manReference      PreProc
 highlight default link manSubHeading     Function
 
+if &filetype != 'man'
+  " Loaded by the location list TOC
+  finish
```

Comment:
Is this because calling `man#show_toc()` triggers some other code that loads ft=man? Or why?